corehq/motech/repeaters/templates/repeaters/bootstrap3/repeat_record_report.html

```diff
@@ -79,6 +79,7 @@ <h4 class="modal-title">{% blocktrans %}Confirmation{% endblocktrans %}</h4>
     {% registerurl 'requeue_repeat_record' domain %}
     {% initial_page_data 'payload_id' payload_id %}
     {% initial_page_data 'repeater_id' repeater_id %}
+    {% initial_page_data 'state' state %}
```

Comment:
You could grab this in js from the filter, right? I don't feel strongly about this, but I was a little surprised to see it here.

Reply:
Same comment for `total`, although it's probably a bit grosser to parse the total out of the table.

Reply:
I was mainly following how it was done for payload_id and repeater_id with regards to state. The page gets re-rendered any time we apply new filters, so this felt easy enough. And generally like the idea of being handed the value rather than inferring it from the UI